apps/bfd-server/bfd-server-docs/src/test/java/gov/cms/bfd/datadictionary/util/FhirElementStreamTest.java

```diff
@@ -0,0 +1,19 @@
+package gov.cms.bfd.datadictionary.util;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Avoid star imports (and adjust your IDE to avoid them) please. (Here and some other files)